logrotate.spec.in

```diff
@@ -47,4 +46,3 @@ touch $RPM_BUILD_ROOT/%{_localstatedir}/lib/logrotate.status
 %attr(0755, root, root) %{_sysconfdir}/cron.daily/logrotate
 %attr(0644, root, root) %config(noreplace) %{_sysconfdir}/logrotate.conf
 %attr(0755, root, root) %dir %{_sysconfdir}/logrotate.d
-%attr(0644, root, root) %verify(not size md5 mtime) %config(noreplace) %{_localstatedir}/lib/logrotate.status
```

Comment:
In my opinion, the state file should be owned by the logrotate RPM package, so should the directory.  Otherwise, it will provoke various audit tools that do not like files/directories in /usr owned by nobody.